core-web/apps/dotcms-ui/src/stories/primeng/table/Table.stories.ts

```diff
@@ -0,0 +1,154 @@
+import { Meta, Story, moduleMetadata } from '@storybook/angular';
+
+import { CommonModule } from '@angular/common';
+
+import { ButtonModule } from 'primeng/button';
+import { TableModule } from 'primeng/table';
+import { TagModule } from 'primeng/tag';
+
+// Get the current date
+const currentDate = new Date();
+
+// Function to add a day to a date
+const addDays = (date, days) => {
+    const result = new Date(date);
+    result.setDate(result.getDate() + days);
+
+    return result;
+};
+
+interface File {
```

Comment:
a File usually is a name or path and some meta info.
This definition looks a bit more stuff, probably a diff name would be good

Reply:
Must to rename this interface

`File` is a existing interface inside Typescript
When we create a interface that alredy exist, both interfaces are merged
So this File interface have all attributes of File (binarys) like `lastModified, code, buffer, etc`